From 324b874cbadcf0d9a7f9187af51008f8efb2f9c1 Mon Sep 17 00:00:00 2001
From: Antoine Martin <dev@ayakael.net>
Date: Tue, 22 Feb 2022 20:07:56 +0000
Subject: [PATCH 1/1] aspnetcore-fix

 Somewhere along the way, installer forgets MicrosoftAspNetCoreApp-
 RuntimePackageVersion, thus expects version 5.0.14-servicing when 
 it actual file is for 5.0.14 This reminds installer what version
 AspNetCoreappRuntime is by re-setting it as what it is usually set.

---
 src/redist/targets/GenerateLayout.targets | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/redist/targets/GenerateLayout.targets b/src/redist/targets/GenerateLayout.targets
index 8e50a8ff7..4da6cef95 100644
--- a/src/redist/targets/GenerateLayout.targets
+++ b/src/redist/targets/GenerateLayout.targets
@@ -11,6 +11,7 @@
 
   <PropertyGroup>
     <!-- Blob storage directories are not stabilized, so these must refer to a package that does not stabilize -->
+    <MicrosoftAspNetCoreAppRuntimePackageVersion>$(MicrosoftNETCoreDotNetHostResolverPackageVersion)</MicrosoftAspNetCoreAppRuntimePackageVersion>
     <AspNetCoreBlobVersion>$(MicrosoftAspNetCoreAppRuntimePackageVersion)</AspNetCoreBlobVersion>
     <CoreSetupBlobVersion>$(MicrosoftNETCoreAppRuntimeVersion)</CoreSetupBlobVersion>
     <WindowsDesktopBlobVersion>$(MicrosoftWindowsDesktopAppPackageVersion)</WindowsDesktopBlobVersion>
-- 
2.34.1

